src/backtrace.rs

```diff
@@ -0,0 +1,56 @@
+use std::{collections::HashSet, path::Path};
+
+use object::read::File as ElfFile;
+use probe_rs::{config::RamRegion, Core};
+
+use crate::{Outcome, VectorTable};
+
+mod pp;
+mod symbolicate;
+mod unwind;
+
+pub(crate) struct Settings<'p> {
+    pub(crate) current_dir: &'p Path,
+    pub(crate) max_backtrace_len: u32,
+    pub(crate) force_backtrace: bool,
+}
+
+/// (virtually) unwinds the target's program and prints its backtrace
+pub(crate) fn print(
```

Comment:
I know I am late to the party, but why are you using `pub(crate)` although it is a application and not a library?